pkg/virt-handler/vm.go

```diff
@@ -227,8 +218,17 @@ func (d *VirtualMachineController) startDomainNotifyPipe(vmi *v1.VirtualMachineI
 			}
 
 			go func(vmi *v1.VirtualMachineInstance) {
-				log.Log.Object(vmi).Infof("Accepted new notify pipe connection for vmi")
 				defer fd.Close()
+
+				// pipe the VMI domain-notify.sock to the virt-handler domain-notify.sock
+				// so virt-handler receives notifications from the VMI
+				conn, err := net.Dial("unix", filepath.Join(d.virtShareDir, "domain-notify.sock"))
+				if err != nil {
+					log.Log.Reason(err).Error("error connecting to domain-notify.sock for proxy connection")
```

Comment:
we arent allowed to continue if err != nil maybe ?
see please the panic
```
{"component":"virt-handler","kind":"","level":"info","msg":"Received Domain Event of type ERROR","name":"","pos":"server.go:78","timestamp":"2020-07-13T23:08:37.889320Z","uid":""}
{"component":"virt-handler","level":"error","msg":"error connecting to domain-notify.sock for proxy connection","pos":"vm.go:227","reason":"dial unix /var/run/kubevirt/domain-notify.sock: connect: no such file or directory","timestamp":"2020-07-13T23:08:37.890808Z"}
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0x18d7bb1]

goroutine 3782 [running]:
kubevirt.io/kubevirt/pkg/virt-handler.(*VirtualMachineController).startDomainNotifyPipe.func1.1(0x37ffc40, 0xc000dac8d8, 0xc000256640, 0xc003388800)
	pkg/virt-handler/vm.go:229 +0x191
created by kubevirt.io/kubevirt/pkg/virt-handler.(*VirtualMachineController).startDomainNotifyPipe.func1
	pkg/virt-handler/vm.go:220 +0x70
```

Reply:
yeah, there needs to be a return statement there. I'll fix that